### PR TITLE
This is from Mohammad Edrees

### PR DESCRIPTION
**Pull request by Edreessaadat**
@Abdulmatin1  please merge it

### CODE REVIEW

**Review by matin-Walizada — APPROVED** [review]

**Comment by matin-Walizada** [comment]
@Edreessaadat  reviewed and approved

**Comment by matin-Walizada** [comment]
aproved

**Review by matin-Walizada — APPROVED** [review]

**Comment by Edreessaadat** [comment]
Thanks

**Review by matin-Walizada — APPROVED** [review]

**Comment by matin-Walizada** [comment]
aproved

File: Core-Java-Assignments/src/com/Assignment_Group_A/Group_A.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 	public void gitHub() {
 		
-		System.out.println("this is from Basir ");
```
Comment (matin-Walizada) [inline Core-Java-Assignments/src/com/Assignment_Group_A/Group_A.java:7]: reviewing your codes

File: Core-Java-Assignments/src/com/Assignment_Group_A/Group_A.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 	public void gitHub() {
 		
-		System.out.println("this is from Basir ");
+		System.out.println("this is from Mohammad Edrees ");
```
Comment (matin-Walizada) [inline Core-Java-Assignments/src/com/Assignment_Group_A/Group_A.java:7]: reviewing your code

**Review by matin-Walizada — APPROVED** [review]

**Comment by matin-Walizada** [comment]
reviewd

**Review by matin-Walizada — APPROVED** [review]

**Comment by matin-Walizada** [comment]
approved

**Comment by matin-Walizada** [comment]
a

**Comment by matin-Walizada** [comment]
approved

**Comment by matin-Walizada** [comment]
approved

**Comment by matin-Walizada** [comment]
approved

**Review by matin-Walizada — APPROVED** [review]

**Review by matin-Walizada — APPROVED** [review]

**Review by matin-Walizada — APPROVED** [review]

**Comment by matin-Walizada** [comment]
approved

**Pull request closed without merging**